dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.References.targets

```diff
@@ -9,7 +9,7 @@
     <_MrtCoreDllDir>$(MSBuildThisFileDirectory)..\runtimes\win10-$(_MrtCoreRuntimeIdentifier)\native\</_MrtCoreDllDir>
 
     <_AddMrtCoreAssembliesToReferenceCopyLocalPaths Condition="'$(_AddMrtCoreAssembliesToReferenceCopyLocalPaths)'=='' AND '$(Platform)' == 'AnyCPU'">false</_AddMrtCoreAssembliesToReferenceCopyLocalPaths>
-    <_AddMrtCoreAssembliesToReferenceCopyLocalPaths Condition="'$(_AddMrtCoreAssembliesToReferenceCopyLocalPaths)'=='' AND '$(ProjectReunionFrameworkPackage)' == 'true'">false</_AddMrtCoreAssembliesToReferenceCopyLocalPaths>
+    <_AddMrtCoreAssembliesToReferenceCopyLocalPaths Condition="'$(_AddMrtCoreAssembliesToReferenceCopyLocalPaths)'=='' AND '$(ProjectReunionMRTCore)' != 'false'">false</_AddMrtCoreAssembliesToReferenceCopyLocalPaths>
```

Comment:
>$(ProjectReunionMRTCore)' != 'false'" [](http://example.com/codeflow?start=124&length=37)

logically this would be "'$(ProjectReunionInApp)' == 'true'", even in reality when "'$(ProjectReunionMRTCore) == 'false'", $(ProjectReunionInApp) would always be true. OTOH, it's not clear to me what value it provides to distinguish the deployment type (reunion or standalone).

Reply:
ProjectReunionInApp no longer exists, and we should avoid reinventing it for just this special purpose. We can call the property whatever we like, as long as it's distinct from ProjectReunionFrameworkPackage, which can be true or false but only with Reunion deployment.  For standalone (this scenario) we need a third state (property).

Reply:
that's a made-up prop. As you are adding another implicit prop, I am wondering what's the collision you mentioned for "ProjectReunionFrameworkPackage != false" here?

Reply:
The collision that existed previously, which InApp support had to work around, was this:  
With ProjectReunionFrameworkPackage set to 'false', the condition here would fail (comparing to 'true') and fall through to binplacing dlls, which collides with InApp support.  InApp worked around that by setting _AddMrtCoreAssembliesToReferenceCopyLocalPaths to 'false' explicitly.  

To clean this up and remove the workaround, we need a distinct check for your inner loop tests.  ProjectReunionFrameworkPackage is no longer set to 'true', so even in Framework Package mode, this check will fail and attempt to binplace.  And in InApp mode, when it's set explicitly to 'false', it will also attempt to binplace.  In both cases, the behavior is wrong.  When in Reunion deployment, we should never binplace.  

So a separate property is needed.

Reply:
Thanks for the explanation! We may make it consistent (with WinUI), and create a separate props file to set it to true (which also provides a convenient place for comment). Also the name would be ProjectReunionFoundation as this is not MRT specific.

Reply:
sounds good - I'll make it happen